.github/workflows/build-nocache.yml

```diff
@@ -0,0 +1,93 @@
+name: Build No Cache
+
+on:
+
+  workflow_dispatch:
+
+  schedule:
+    - cron: '30 12 * * 0'
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_IMAGE: dfedigital/register-trainee-teachers
+      DB_USERNAME: postgres
```

Comment:
Are the DB_* environment variables used in this pipeline?  I can see they're used in the docker-compose steps of the original pipeline but it's not clear where they are used here

Reply:
removed DB env vars